containers/R_Python/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM clipper/py-rpc:latest
+
+## Set a default user. Available via runtime flag `--user docker` 
+## Add user to 'staff' group, granting them write privileges to /usr/local/lib/R/site.library
+## User should also have & own a home directory (for rstudio or linked volumes to work properly). 
+RUN useradd docker \
+	&& mkdir /home/docker \
+	&& chown docker:docker /home/docker \
+	&& addgroup docker staff
+
+RUN apt-get update \ 
+	&& apt-get install -y --no-install-recommends \
+		ed \
+		less \
+		locales \
+		vim-tiny \
+		wget \
+		ca-certificates \
+		fonts-texgyre \
```

Comment:
Some of these packages seem unnecessary. Why are you installing ed, less, vim-tony, fonts-texgyre?